Prog/PresStudyTabulations.sas

```diff
@@ -0,0 +1,369 @@
+    ( all='DC Total' ward2022=' ' )
+    ,
+    /** Columns **/
+    n='Projects' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  table 
+    /** Rows **/
+    ( all='DC Total' ward2022=' ' )
+    ,
+    /** Columns **/
+    sum='Assisted Units' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  format ProgCat ProgCat.;
+run;
+
+title3 "Project and assisted unit unique counts by ANC";
+
+proc tabulate data=Project_assisted_units format=comma10. noseps missing;
+  where ProgCat ~= . and not( missing( anc2012 ) );
+  class ProgCat / preloadfmt order=data;
+  class anc2012;
+  var mid_asst_units err_asst_units;
+  table 
+    /** Rows **/
+    ( all='DC Total' anc2012=' ' )
+    ,
+    /** Columns **/
+    n='Projects' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  table 
+    /** Rows **/
+    ( all='DC Total' anc2012=' ' )
+    ,
+    /** Columns **/
+    sum='Assisted Units' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  format ProgCat ProgCat.;
+run;
+
+title3 "Project and assisted unit unique counts by Neighborhood Custer";
+
+proc tabulate data=Project_assisted_units format=comma10. noseps missing;
+  where ProgCat ~= . and not( missing( cluster2017 ) );
+  class ProgCat / preloadfmt order=data;
+  class cluster2017;
+  var mid_asst_units err_asst_units;
+  table 
+    /** Rows **/
+    ( all='DC Total' cluster2017=' ' )
+    ,
+    /** Columns **/
+    n='Projects' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  table 
+    /** Rows **/
+    ( all='DC Total' cluster2017=' ' )
+    ,
+    /** Columns **/
+    sum='Assisted Units' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  format ProgCat ProgCat.;
+run;
+
+****Creat Data set for Age of Building Table***;
+data Project_AOB;
+
+  merge
+    Project_assisted_units
+		(in=inProject)
+    Prescat.Parcel
+		(in=inParcel);
+  by NLIHC_ID;
+  	if inProject and inParcel;
+   
+run;
+*****Errors Start somewhere around here******;
+
+data Project_Age_Of_Building;
+
+  merge
+   Project_AOB
+   	(in=inProject)
+   Realprop.Cama_parcel_2023_05
+   	(in=inCama);
+
+  by SSL;
+ 	 if inProject and inCama;
+   
+run;
+
+proc summary data=Project_Age_Of_Building 
+  by nlihc_id;
+  class nlihc_id portfolio;
+  output out=Project_Building_Age
+    min(AYB)=;
+run;
+****Age of Building Table***;
+proc tabulate data=Project_Building_Age format=comma10. noseps missing;
+  where ProgCat ~= . and not( missing( AYB ) );
+  class ProgCat / preloadfmt order=data;
+  class AYB;
+  var mid_asst_units err_asst_units;
+  table 
+    /** Rows **/
+    ( all='DC Total' AYB=' ' )
+    ,
+    /** Columns **/
+    n='Projects' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  table 
+    /** Rows **/
+    ( all='DC Total' AYB=' ' )
+    ,
+    /** Columns **/
+    sum='Assisted Units' * ( all='\b Total' ProgCat=' ' ) * mid_asst_units=' '
+    ;
+  format ProgCat ProgCat.;
+  format value year_built (notsorted)
```

Comment:
`ERROR 22-322: Syntax error, expecting one of the following: a name, a format name, ...`

The code to create a new user format needs to be a Proc Format (not a Format statement) which is separate from and comes before the Proc Tabulate code. Please refer back to the code I included in the issue. You then assign the year_built format you created to the AYB variable using a Format statement as part of the Proc Tabulate, such as is being done with `format ProgCat ProgCat.;`.

[You can look here](https://www.statology.org/sas-proc-format/) for other examples of using Proc Format.